app/views/announcements/_announcement.html.erb

```diff
@@ -0,0 +1,8 @@
+  <% if announcement_exists?(current_announcement) %>
```

Comment:
This seems an unnecessary abstraction

Reply:
You can just wrap it in 
```erb
<% if current_announcemrnt %>
<% end %>
```

Also allow current_announcement to be nil

Reply:
I tried but ending with errors..
![2020-10-14_23-02](https://user-images.githubusercontent.com/55848322/96024555-81ff0800-0e71-11eb-854b-41d160911ada.png)

Reply:
It is because of this line
```ruby
     order("created_at desc").first || new 
```

Remove `|| new` part

Reply:
ohk thnks sir worked fine !!!